File: lib12/src/test/java/TestTest.java

import org.junit.jupiter.api.Test;

import java.util.Random;

import static org.junit.jupiter.api.Assertions.assertTrue;

class TestTest {

    @Test
    void test() throws InterruptedException {
        Thread.sleep(20 + Math.abs(new Random().nextLong() % 1000));
        assertTrue(true);
    }
}

File: lib3/src/test/kotlin/TestTest.kt
import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test
import java.util.*

class TestTest {

    @Test
    fun test(){
        Thread.sleep(20 + Math.abs(Random().nextLong() % 1000))
        assertTrue(true)
    }
}